Modules/Quotas/test/Quotas.Application.Tests/Tests/Identities/QuotaCreatedForTierIntegrationEventHandlerTests.cs

```diff
@@ -43,9 +45,38 @@ public async void Creates_tier_quota_after_consuming_integration_event()
         ).MustHaveHappened();
     }
 
+    [Fact]
+    public async void Updates_metric_statuses_after_creating_tier_quota()
+    {
+        // Arrange
+        var tierId = new TierId("TIRFxoL0U24aUqZDSAWc");
+        var tier = new Tier(tierId, "some-tier-name");
```

Comment:
This is never used or rather never has to be used (its only usage is `tier.Id`, which can be replaced with `tierId`.